subprojects/packagefiles/xtensor/meson_options.txt

```diff
@@ -0,0 +1,6 @@
+option('XTENSOR_USE_XSIMD', type: 'boolean', value: false, description: 'simd acceleration for xtensor')
```

Comment:
we don't usually use capslock when writing options, unlike cmake.

Reply:
Also please remove the project name from the option.

When used as a subproject, the current name of the option is
```
-D xtensor:XTENSOR_USE_XSIMD=false
```